README.md

```diff
@@ -38,8 +62,7 @@ a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the
 
         pip install -U tensorflow
 
-3. Checkout `v0.6.1` from the `ngraph-tf` repo and build the bridge
-   as follows: 
+3. Checkout `v0.6.1` from the `ngraph-tf` repo and build it: 
    
```

Comment:
the PR name says `Changes for r0.7`, but we have `v0.6.1` in many places.
will we be updating the readme once during 0.7 release? in that case all the 0.6.1's need to be 7